source/templates/code_signing.slim

```diff
@@ -302,27 +302,27 @@
 							p
 								span> == data[:strings][:code_signing][:android_keystore][:keystore_password_as]
 								code.env-var[ng-bind="KeystoreFile.androidKeystorePasswordEnvVarKey | prettifiedKey"]
-							input.env-var[type="text" ng-if="androidKeystoreCtrl.showPassword" name="password" required ng-model="androidKeystoreCtrl.keystoreFile.password" placeholder="#{ data[:strings][:code_signing][:android_keystore][:password_placeholder] }"]
+							input.env-var[type="text" ng-if="androidKeystoreCtrl.showPassword" name="password" ng-required="androidKeystoreCtrl.keystoreFile.password" ng-model="androidKeystoreCtrl.keystoreFile.password" placeholder="#{ data[:strings][:code_signing][:android_keystore][:password_placeholder] }"]
 							input.env-var[type="text" ng-if="!androidKeystoreCtrl.showPassword" ng-model="codeSigningCtrl.placeholderForText(androidKeystoreCtrl.keystoreFile.password)" ng-model-options="{getterSetter: true}" disabled]
-							.show-password
+							.show-password[ng-if="androidKeystoreCtrl.keystoreFile.password"]
```

Comment:
It should not be this way. It should be visible always. Whenever the file loads (or has been uploaded), toggle it on based on whether the input is empty or not. And whenever the fields have been saved, do the same toggling with the same condition.